File: FeedItForward/frontend/src/schemas/cssMessage.ts
export interface CSSMessage {
  css_message_id: number;

  text: string;
  sender_user_id: number;
  receiver_user_id: number;
  datetime: string;

  css_history_id: number;
}

export interface CSSMessageCreate {
  text: string;
  sender_user_id: number;
  receiver_user_id: number;

  css_history_id: number;
}

File: FeedItForward/backend/app/schemas/css_message.py
from pydantic import BaseModel

class CSSMessage(BaseModel):
    css_message_id: int

    text: str
    sender_user_id: int
    receiver_user_id: int
    datetime: str

    css_history_id: int

class CSSMessageCreate(BaseModel):
    text: str
    sender_user_id: int
    receiver_user_id: int

    css_history_id: int
